.github/workflows/ci.yml

```diff
@@ -216,8 +216,7 @@ jobs:
             CppUnit::TestCaller<UniqueExpireLRUCacheTest>.testExpireN,
             CppUnit::TestCaller<ExpireLRUCacheTest>.testAccessExpireN,
             CppUnit::TestCaller<SyslogTest>.testOldBSD,
-            CppUnit::TestCaller<PollSetTest>.testPollClosedServer,
-            CppUnit::TestCaller<CryptoTest>.testEncryptDecryptGCM"
+            CppUnit::TestCaller<PollSetTest>.testPollClosedServer"
```

Comment:
this should be looked into when working on #4382; not being able to detect the server closing connection is a problem